### PR TITLE
Moving Carousel and Tabs out of the sandbox

### PR DESCRIPTION
**Pull request by vladbailescu**

### CODE REVIEW

**Comment by vladbailescu** [comment]
A few things to review:
* location and versioning system for newly introduced commons clientlibs and component
* location (and possible versioning) for childreneditor Sling Models; also adding a `package-info`
* removed `sandbox` completely as the whole `development` branch can be considered a sandbox

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/ui/childreneditor/Item.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
-package com.adobe.cq.wcm.core.components.sandbox.ui.childreneditor;
+package com.adobe.cq.wcm.core.components.ui.childreneditor;
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/ui/childreneditor/Item.java]: `editor` instead of `ui`?

Comment (vladbailescu) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/ui/childreneditor/Item.java]: I was also considering `helpers`

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/ui/childreneditor/Editor.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
 import com.day.cq.wcm.api.components.ComponentManager;
 
 /**
- * Defines an {@code Editor} Sling Model used by the {@code /apps/core/wcm/sandbox/commons/ui/components/childreneditor} UI component.
+ * Defines an {@code Editor} Sling Model used by the {@code /apps/core/wcm/components/commons/editor/components/childreneditor/v1/childreneditor} UI component.
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/ui/childreneditor/Editor.java]: Why not keeping `ui`?

**Pull request merged**